src/components/inputv2/inputv2.js

```diff
@@ -54,10 +54,29 @@ const getBorderForState = state => {
   if (state.isDisabled) {
     return borderColorDisabled;
   }
+  if (state.isHovered) {
+    return state.isInvalid // eslint-disable-line
+      ? colors.solidColors.redv2
+      : state.isDisabled
+      ? borderColorDisabled
+      : borderColorHover;
+  }
 
   return borderColorDefault;
 };
 
+const getLabelColor = state => {
+  if (state.isHovered) {
+    return colors.greys.grey1;
```

Comment:
This won't work for both themes (light + dark). Would need to add two new constants like

```
const labelColorHovered = theme('mode', {
  light: colors.greys.grey1,
  dark: ?
});

const labelColorFocused = theme('mode', {
  light: colors.greys.grey3,
  dark: ?
});
```

Then return one of those from `getLabelColor`